File: Modul-4/Soal 3/PRAK403-2410817110009-MuhammadAlfiGunawan.py
a = int(input("Masukkan Angka Pertama: "))
b = int(input("Masukkan Angka Kedua: "))

x = a
y = b
while True:
    print(x, y, end="")

    if(x == b and y == a): break

    if(a > b):
        x-=1
        y+=1
    else: 
        x+=1
        y-=1
    
    print(" - ", end="")

File: Modul-4/Soal 3/PRAK403-2410817110009-MuhammadAlfiGunawan.c
#include <stdio.h>

int main(){
    int a, b;
    printf("Masukkan Angka Pertama: ");
    scanf("%d", &a);
    printf("Masukkan Angka Kedua: ");
    scanf("%d", &b);

    int x = a, y = b;
    while(1){ 
        printf("%d %d", x, y);
        if (x == b && y == a) 
        break; 

        if(a > b){
            x--;
            y++;
        } 
        else{
            x++;
            y--;
        }
        printf(" - ");
    }
}
